Header header
int32 ID
float32 translation
float32 rotation

